pkg/provider/provider.go

```diff
@@ -100,10 +100,11 @@ func ConvertArrayPods(pods []*corev1.Pod) (out []*Pod) {
 }
 
 type TestEnvironment struct { // rename this with testTarget
-	Namespaces        []string               `json:"testNamespaces"`
-	Pods              []*Pod                 `json:"testPods"`
-	Containers        []*Container           `json:"testContainers"`
-	Operators         []Operator             `json:"testOperators"`
+	Namespaces        []string     `json:"testNamespaces"`
+	Pods              []*Pod       `json:"testPods"`
+	Containers        []*Container `json:"testContainers"`
+	Operators         []Operator   `json:"testOperators"`
+	PersistentVolumes []corev1.PersistentVolume
```

Comment:
Don't you need to get the `PersistentVolumes` from the autodiscover package inside `buildTestEnvironment()`?

Reply:
Good catch.  Fixed!